extensions/content-manager/controllers/collection-types.js

```diff
@@ -0,0 +1,342 @@
+'use strict';
```

Comment:
Is this file auto-generated?

Reply:
yeah

Reply:
This file is the file that I use to make it so when they create a project through the strapi dashboard it does all the google stuff.